RocketChat/Rocket.Chat
# [FIX] Modifying userActionIndicator syntax for translations

## Proposed changes (including videos or screenshots)
Replaced `is + action`/`are + action` strings with `is_action` and `are_action` strings, for instance:
- `is_typing` / `are_typing` ✅ (already exist in `i18n.json` files)
- `is_recording` / `are_recording` 🚧 (needs to be added to translations)
- `is_uploading` / `are_uploading` 🚧 (needs to be added to translations)

With that, the wrong French translation "*is* en train d'écrire" becomes "*est* en train d'écrire", and **it is also possible to correctly translate languages not using "is", like German**

![image](https://user-images.githubusercontent.com/18146038/193936678-e094ed48-50a2-4d49-8ac3-f53376f5f561.png)

- It also gives flexibility for future actions** 😄 
- Fixes bad translations about user(s) activity
This bug was been introduced by #22392







## Issue(s)

Fixes #23593
Fixes #24916
Fixes #25045
Fixes #26243

## Steps to test or reproduce

Open multiples browsers/windows with different users, and type/upload/record on one (or many) of them to see the activity test above the chat input

## Further comments

Example of language file with this fix, for French
`apps/meteor/packages/rocketchat-i18n/i18n/fr.i18n.json`
```diff
+  "are_recording": "sont en train d'enregistrer",
"are_typing": "sont en train d'écrire",
+  "are_uploading": "sont en train de mettre en ligne",
[...]
+  "is_recording": "est en train d'enregistrer",
"is_typing": "est en train d'écrire",
+  "is_uploading": "est en train de mettre en ligne",
```
